src/test/java/org/casbin/jcasbin/main/EnforcerUnitTest.java

```diff
@@ -21,13 +21,19 @@
 
 import java.io.FileInputStream;
 import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 
 import static java.util.Arrays.asList;
 import static org.casbin.jcasbin.main.CoreEnforcer.newModel;
 import static org.casbin.jcasbin.main.TestUtil.testEnforce;
 import static org.casbin.jcasbin.main.TestUtil.testGetPolicy;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
+import static org.casbin.jcasbin.util.Util.LOGGER;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
```

Comment:
Why again this import? BTW, you should just import ``org.junit.Assert`` if you have used many names under this scope.

Reply:
> Why again this import?
Are you just referring to Assert? or is LOGGER also included in this comment?
If it's just Assert then I fixed it

Reply:
> Are you just referring to Assert? or is LOGGER also included in this comment?
If it's just Assert then I fixed it

Just Assert